en/config/compass.md

```diff
@@ -42,6 +48,17 @@ The calibration steps are:
    
 Once you've calibrated the vehicle in all the positions *QGroundControl* will display *Calibration complete* (all orientation images will be displayed in green and the progress bar will fill completely). You can then proceed to the next sensor. 
 
+### Partial "Quick" calibration
+
+This calibration is similar to the well-known figure-8 compass calibration done on a smartphone.
+
+1. Hold the vehicle in front of you and randomly perform partial rotations on all its axes.
+1. Wait for the heading estimate to stabilize and verify that the compass rose is now pointing to the correct direction (this can take a couple of seconds).
+
+Notes:
+- There is no start/stop for this type of calibration, the algorithm is running all the time when the vehicle is disarmed.
+- The calibration is immediately applied to the data (no reboot is required) but is saved to the calibration parameters after disarming the vehicle only (the calibration is lost if no arming/disarming sequence is performed between calibration and shutdown).
+- The amplitude and the speed of the partial rotations done in step 1 can affect the calibration quality. However, 2-3 oscillations of ~30 degrees in every direction is usually enough.
```

Comment:
```suggestion
This calibration is similar to the well-known figure-8 compass calibration done on a smartphone:

1. Hold the vehicle in front of you and randomly perform partial rotations on all its axes.
1. Wait for the heading estimate to stabilize and verify that the compass rose is pointing to the correct direction (this can take a couple of seconds).

Notes:

- There is no start/stop for this type of calibration (the algorithm runs continuously when the vehicle is disarmed).
- The calibration is immediately applied to the data (no reboot is required) but is saved to the calibration parameters after disarming the vehicle only (the calibration is lost if no arming/disarming sequence is performed between calibration and shutdown).
- The amplitude and the speed of the partial rotations done in step 1 can affect the calibration quality.
  However, 2-3 oscillations of ~30 degrees in every direction is usually enough.
```